benchmarks/cloudsuite3/graph-analytics/run.sh

```diff
@@ -138,25 +132,28 @@ done
 rm -rf results
 mkdir -p results
 
-run_setup
+docker create --name graph-data cloudsuite3/twitter-dataset-graph
 
+echo "Test: docker run $PRIVILEGED $MAXMEM $MAXSWAP --ulimit nofile=90000:90000 $CPUSETS_CPU $CPUSETS_MEM --rm --volumes-from graph-data $CONTAINER $DMEM $EMEM"
 start_time=$(date +%s%N)
 docker run $PRIVILEGED $MAXMEM $MAXSWAP --ulimit nofile=90000:90000 $CPUSETS_CPU $CPUSETS_MEM --rm --volumes-from graph-data $CONTAINER $DMEM $EMEM > results/raw_results.txt
 end_time=$(date +%s%N)
 time_taken=$(echo "scale=9; ($end_time - $start_time)/1000000000" | bc)
 
-echo "graph-analytics,\"$NOTE\",$time_taken," `grep time results/raw_results.txt | sed 's/Running time = //'` >> $RESULTS
+docker rm graph-data
```

Comment:
Ideally, we would like to verify this command succeeds before proceeding and handle any errors appropriately.